stix2slider/convert_pattern.py

```diff
@@ -513,22 +516,22 @@ def add_list_file_property_pattern(file_obj, properties, rhs, op, id20):
 def add_hashes_pattern(obj, hash_type, rhs, op, id20):
     if hash_type == "MD5":
         obj.md5 = rhs.value
-        convert_operator(op, obj.hashes._hash_lookup(Hash.TYPE_MD5), id20)
+        convert_operator(op, obj.hashes._hash_lookup(Hash.TYPE_MD5).simple_hash_value, id20)
```

Comment:
When similar lines of code need to be changed in similar ways, that's a sign to me that the code could be refactored. In this case, `setattr` could be used to account for the different attribute names for the hash attributes. Or you could use the `_set_hash()` function directly (passing the hash type that way)

Reply:
can you give example code?

Reply:
It's a bit tricky, because of different styles of specifying hash types, but (roughly):

```
cybox2_hash_type = hash_type.replace('-', '')
attr_name = cybox2_hash_type.lower()
setattr(obj, attr_name, rhs.value)
convert_operator(op, obj.hashes._hash_lookup(cybox2_hash_type).simple_hash_value, id20)
```

If the first two lines don't work (there are some exceptions), you can just do a dictionary lookup to map the types from STIX 2 to STIX1/CybOX2.

Reply:
Done